app/library/postgres.py

```diff
@@ -72,9 +73,13 @@ async def get_asset_from_postgres(asset_id: str):
 
 async def delete_asset_from_postgres(asset_id: str):
     # TODO: delete from postgres
+    try:
+        delete_asset(asset_id)
+    except Exception as e:
+        raise e
```

Comment:
Let's do `delete_asset(asset_id)` just as is here, so we don't slow down the response by catching the exception and then propagating it.
Alternatively, if we want to make this more DCC friendly, we should catch it and turn it human readable:
```
try:
        delete_asset(asset_id)
    except psycopg2.errors.ForeignKeyViolation as e:
        raise AssertionError("One or more scenes are referencing this asset, please delete the scene before deleting this asset")
```